packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -26,6 +26,7 @@ type TTradeModalProps = {
     derivez_tokens: TCFDDashboardContainer['derivez_tokens'];
     is_demo: string;
     platform: TCFDsPlatformType;
+    is_mobile?: boolean;
 };
 
 const PlatformIconsAndDescriptions = (
```

Comment:
This component itself can take observer, no need to pass props for is_mobile

Reply:
same here